pkg/gui/controllers/helpers/commits_helper.go

```diff
@@ -19,10 +19,9 @@ type ICommitsHelper interface {
 type CommitsHelper struct {
 	c *HelperCommon
 
-	getCommitSummary     func() string
-	setCommitSummary     func(string)
-	getCommitDescription func() string
-	// getVerifyFlag                 func() bool
+	getCommitSummary              func() string
+	setCommitSummary              func(string)
+	getCommitDescription          func() string
```

Comment:
This should go into the previous commit, I suppose. Also below.

Reply:
True. I kind of expected you would squash on merge 😄

Reply:
No, we don't do that here. On the contrary, I value clean and well-structured commit histories very much. (Not just for the end result of a branch when it is merged, but also during review, so I prefer a [fixup-based workflow](https://github.com/jesseduffield/lazygit/blob/master/docs/Fixup_Commits.md) over just piling commits onto one another, which is what we are now doing here. Something to keep in mind for next time, it's too late for this one. 😄)